src/components/ordval/guessOption.tsx

```diff
@@ -92,16 +91,38 @@ export const OrdvalGuessOption = ({
       opacity={showSolution ? (solution?.id === guess.id ? 1 : 0.5) : undefined}
       position="relative"
       w="full"
-      h={{ base: "16", md: "32", lg: "40" }}
       {...props}
     >
+      <Image
+        src={guess.pImage ?? guess.image}
+        alt={guess.desc}
+        objectFit="contain"
+        w="full"
+        h={showSolution ? { base: "25vh", md: "72", lg: "96" } : 0}
+        loading="lazy"
+        pointerEvents="none"
+        blendMode="darken"
+        visibility={showSolution ? 'visible' : 'hidden'}
+      />
+      <Box
+        display={showSolution ? 'none' : 'block'}
+        backgroundColor="gray.50"
+        filter="auto"
+        blur="10px" w="80%"
+        m="0 auto"
+        h={{ base: "28vh", md: "80", lg: "28rem" }}
+      />
       {showConfetti && (
         <Center position="absolute" top="0" bottom="0" left="0" right="0">
           <ConfettiExplosion colors={CONFETTI_COLORS} height="80vh" />
         </Center>
       )}
 
       <Text
+        position={showSolution ? 'inherit' : 'absolute'}
+        top={showSolution ? "0" : { base: "10vh", md: "28", lg: "40" }}
```

Comment:
Can put bottom here instead, won't need to be conditional

Reply:
I did this to follow this picture:
![image](https://github.com/avikantz/ikea-word-games/assets/1447119/c851814d-72c6-46ba-80bb-310fe935f1b5)
but anyway I will put it down, this will simplify a lot..

Reply:
I'd quickly put a <Center> to wrap the text around to see